internal/tofu/node_resource_abstract.go

```diff
@@ -220,7 +257,12 @@ func (n *NodeAbstractResource) RootReferences() []*addrs.Reference {
 			continue
 		}
 
-		refs, _ := lang.ReferencesInExpr(addrs.ParseRef, importTarget.Config.ID)
+		refs, _ := referencesInImportAddress(importTarget.Config.To)
+
+		// TODO - Add RootReferences of ForEach here later one, once for_each is added
```

Comment:
Is this coming in another PR?

Reply:
Yes, it'll be included in the next PR, supporting the ForEach syntax for the import block.